test/test_fill_database.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+import unittest
+import logging
+import re
+from datetime import date
+
+import sys, os
+try:
+    sys.path.append(os.path.join(os.environ['SPARK_HOME'], "python"))
+except KeyError:
+    print "SPARK_HOME not set"
+    sys.exit(1)
+import pyspark
+
+import dataset
+
+import sys, os
+from crash_rate_aggregates.fill_database import compare_crashes, COMPARABLE_DIMENSIONS, DIMENSION_NAMES
+
+class TestStringMethods(unittest.TestCase):
+    def setUp(self):
+        self.sc = pyspark.SparkContext(master="local[1]")
+        self.raw_pings = self.sc.parallelize(list(dataset.generate_pings()))
+
+        self.crash_rate_aggregates = compare_crashes(
+            self.raw_pings,
+            date(2016, 3, 5), date(2016, 6, 7),
```

Comment:
Could you read the dates directly from `dataset.py`? It would be nice to have a single definitions for all those dates.